editor/src/core/shared/element-path-tree.ts

```diff
@@ -141,9 +141,16 @@ function getReorderedPaths(
   metadata: ElementInstanceMetadataMap,
   missingParents: ElementPath[],
 ): ElementPath[] {
-  const elementsToReorder = original.filter(
-    (path) => findMaybeConditionalExpression(path, metadata) != null,
-  )
+  const elementsToReorder = original.filter((path) => {
+    const element = MetadataUtils.findElementByElementPath(metadata, path)
+    if (element == null) {
+      return false
+    }
+    return (
+      MetadataUtils.isConditionalFromMetadata(element) ||
+      MetadataUtils.isExpressionOtherJavascriptFromMetadata(element)
+    )
+  })
```

Comment:
(I think the answer is "no", but just to be sure) Does this block need to also check the feature switch?

Reply:
no I don't think so, that reordering for conditionals should just apply transparently in this case as well

Reply:
Not really, because there are expressions are not in the metadata without the feature switch

Reply:
I mean, even if they are not in the metadata this won't be affecting the editors where the FS is off 🤔